src/rpdk/core/generate_canary.py

```diff
@@ -0,0 +1,83 @@
+import json
+import logging
+from pathlib import Path
+
+from .generate_stack_templates import StackTemplateGenerator
+
+TARGET_CANARY_ROOT_FOLDER = "canary-bundle"
+TARGET_CANARY_FOLDER = "canary-bundle/canary"
+RPDK_CONFIG_FILE = ".rpdk-config"
+CANARY_FILE_PREFIX = "canary"
+CONTRACT_TEST_DEPENDENCY_FILE_NAME = "dependencies.yml"
+CANARY_DEPENDENCY_FILE_NAME = "bootstrap.yaml"
+CANARY_SETTINGS = "canarySettings"
+TYPE_NAME = "typeName"
+CONTRACT_TEST_FILE_NAMES = "contract_test_file_names"
+INPUT1_FILE_NAME = "inputs_1.json"
+FILE_GENERATION_ENABLED = "file_generation_enabled"
+
+
+class CanaryFileGenerator:
```

Comment:
having a class for that is an overkill, can we simply use a module with one method that generates files?

Reply:
Relocating this code to the project class and invoking the StackTemplate generator is an alternative approach. However, encapsulating it within a separate class promotes clear logical separation, enhancing code readability and maintainability.